datafusion/functions-aggregate/src/first_last.rs

```diff
@@ -291,6 +296,7 @@ impl AggregateUDFImpl for FirstValue {
     }
 }
 
+// TODO: rename to PrimitiveGroupsAccumulator
```

Comment:
I think this kind of improvement is better to included in the same PR to reduce confusion.